### ISSUE TITLE
Cannot scrape any game art.

### ISSUE THREAD

**Issue by slapjackios** [prosthetichead/GarlicPress#6]
I'm getting this error everytime I press the "Auto Update Art" button.

![image](https://github.com/prosthetichead/GarlicPress/assets/71088560/34890a92-6593-47e6-835e-3038b347049c)

I'm not getting an image file on my SD card, but I can see the scraped image for the game on the assets folder on my GarlicPress.20230717 folder (where I extracted the zip).

![image](https://github.com/prosthetichead/GarlicPress/assets/71088560/22884262-011e-4730-9022-4c067e661b0c)

The error details read:

```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at GarlicPress.MainForm.OverlayImageWithSkinBackground()
   at GarlicPress.MainForm.UpdateArt(Boolean promptName)
   at GarlicPress.MainForm.btnUpdateImg_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
System.Private.CoreLib
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Private.CoreLib.dll
----------------------------------------
GarlicPress
    Assembly Version: 1.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Runtime
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Runtime.dll
----------------------------------------
System.Windows.Forms
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.2023.32104
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.20/System.Windows.Forms.dll
----------------------------------------
System.ComponentModel.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.ComponentModel.Primitives.dll
----------------------------------------
System.Windows.Forms.Primitives
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.2023.32104
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.20/System.Windows.Forms.Primitives.dll
----------------------------------------
System.Runtime.InteropServices
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Runtime.InteropServices.dll
----------------------------------------
System.Drawing.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Drawing.Primitives.dll
----------------------------------------
System.Collections.Specialized
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Collections.Specialized.dll
----------------------------------------
System.Threading
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Threading.dll
----------------------------------------
System.Diagnostics.TraceSource
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Diagnostics.TraceSource.dll
----------------------------------------
System.Collections
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Collections.dll
----------------------------------------
System.Drawing.Common
    Assembly Version: 7.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
AdvancedSharpAdbClient
    Assembly Version: 2.5.6.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Configuration.ConfigurationManager
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.20/System.Configuration.ConfigurationManager.dll
----------------------------------------
System.ObjectModel
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.ObjectModel.dll
----------------------------------------
Microsoft.Win32.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/Microsoft.Win32.Primitives.dll
----------------------------------------
System.ComponentModel.EventBasedAsync
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.ComponentModel.EventBasedAsync.dll
----------------------------------------
System.Threading.Thread
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Threading.Thread.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 6.0.2023.32104
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.20/Accessibility.dll
----------------------------------------
System.ComponentModel.TypeConverter
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.ComponentModel.TypeConverter.dll
----------------------------------------
Microsoft.Win32.SystemEvents
    Assembly Version: 7.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Collections.Concurrent
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Collections.Concurrent.dll
----------------------------------------
System.ComponentModel
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.ComponentModel.dll
----------------------------------------
System.Memory
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Memory.dll
----------------------------------------
System.Runtime.CompilerServices.Unsafe
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Runtime.CompilerServices.Unsafe.dll
----------------------------------------
System.Numerics.Vectors
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Numerics.Vectors.dll
----------------------------------------
System.Resources.Extensions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.20/System.Resources.Extensions.dll
----------------------------------------
System.Drawing
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.2023.32104
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.20/System.Drawing.dll
----------------------------------------
System.Linq
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Linq.dll
----------------------------------------
System.Private.Uri
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Private.Uri.dll
----------------------------------------
System.Runtime.InteropServices.RuntimeInformation
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Runtime.InteropServices.RuntimeInformation.dll
----------------------------------------
System.Diagnostics.Process
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Diagnostics.Process.dll
----------------------------------------
System.Collections.NonGeneric
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Collections.NonGeneric.dll
----------------------------------------
System.Security.Cryptography.Algorithms
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Security.Cryptography.Algorithms.dll
----------------------------------------
System.Security.Cryptography.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Security.Cryptography.Primitives.dll
----------------------------------------
System.Xml.ReaderWriter
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Xml.ReaderWriter.dll
----------------------------------------
System.Private.Xml
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Private.Xml.dll
----------------------------------------
System.Text.RegularExpressions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Text.RegularExpressions.dll
----------------------------------------
System.Net.WebClient
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Net.WebClient.dll
----------------------------------------
System.Text.Encoding.Extensions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Text.Encoding.Extensions.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.dll
----------------------------------------
System.Runtime.Serialization.Formatters
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Runtime.Serialization.Formatters.dll
----------------------------------------
System.Xml.XmlSerializer
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Xml.XmlSerializer.dll
----------------------------------------
System.Net.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Net.Primitives.dll
----------------------------------------
Microsoft.Extensions.Logging.Abstractions
    Assembly Version: 7.0.0.0
    Win32 Version: n/a
    CodeBase: 
----------------------------------------
System.Net.Sockets
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Net.Sockets.dll
----------------------------------------
System.Diagnostics.Tracing
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Diagnostics.Tracing.dll
----------------------------------------
System.Text.Json
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Text.Json.dll
----------------------------------------
System.Reflection.Emit.Lightweight
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Reflection.Emit.Lightweight.dll
----------------------------------------
System.Reflection.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Reflection.Primitives.dll
----------------------------------------
System.Reflection.Emit.ILGeneration
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Reflection.Emit.ILGeneration.dll
----------------------------------------
System.Text.Encodings.Web
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Text.Encodings.Web.dll
----------------------------------------
System.Runtime.Intrinsics
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Runtime.Intrinsics.dll
----------------------------------------
System.Runtime.Loader
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Runtime.Loader.dll
----------------------------------------
System.Diagnostics.StackTrace
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Diagnostics.StackTrace.dll
----------------------------------------
System.Reflection.Metadata
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Reflection.Metadata.dll
----------------------------------------
System.Collections.Immutable
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Collections.Immutable.dll
----------------------------------------
System.Diagnostics.FileVersionInfo
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Diagnostics.FileVersionInfo.dll
----------------------------------------
System.Net.Requests
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Net.Requests.dll
----------------------------------------
System.Net.WebHeaderCollection
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Net.WebHeaderCollection.dll
----------------------------------------
System.Threading.ThreadPool
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Threading.ThreadPool.dll
----------------------------------------
System.Net.Http
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Net.Http.dll
----------------------------------------
System.Web.HttpUtility
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Web.HttpUtility.dll
----------------------------------------
System.Diagnostics.DiagnosticSource
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Diagnostics.DiagnosticSource.dll
----------------------------------------
System.Net.Security
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Net.Security.dll
----------------------------------------
System.Security.Cryptography.X509Certificates
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Security.Cryptography.X509Certificates.dll
----------------------------------------
System.Threading.Overlapped
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Threading.Overlapped.dll
----------------------------------------
System.Net.NameResolution
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Net.NameResolution.dll
----------------------------------------
System.Security.Principal.Windows
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Security.Principal.Windows.dll
----------------------------------------
System.Security.Claims
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Security.Claims.dll
----------------------------------------
System.Security.Cryptography.Encoding
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Security.Cryptography.Encoding.dll
----------------------------------------
System.Net.ServicePoint
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.2023.32017
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.20/System.Net.ServicePoint.dll
----------------------------------------

************** JIT Debugging **************

```

- Windows 11
- stock+garlic_test_20230629_1

**Comment by prosthetichead** [comment]
I think this is also related to the issue with reading the skin file #5 
If the file assets/skinSettings.json can't be read, then it will have a null exception when it tries to work out if the text should be left right or centered on the preview image. will hold on this issue till we can sort out #5

**Comment by prosthetichead** [comment]
Can you try this again with release 20230718? I think it may have been released to issue #5

**Comment by prosthetichead** [comment]
@slapjackios 
Going to close this issue if you're still having this problem please comment and we can reopen it.
